examples/tutorials/tut_00_prerequisites.md

```diff
@@ -12,31 +12,29 @@ If you run Windows, some commands might be different.
 
 ## Download ## {#tut_download}
 
-We use *Mercurial (hg)* for revision control. You need to have it
-installed on your system in order to download *Musubi*.
-Follow [these instructions](http://mercurial.selenic.com/
-"Mercurial website") if you haven't, yet.
-Then, create a directory for everything that happens in these tutorials
+We use *git* with submodules for revision control.
+You need to have it installed on your system in order to download *Musubi*.
+
+Create a directory for everything that happens in these tutorials
 (we will call this directory `apes`, but you can use a different name.
```

Comment:
Please close the bracket, e.g.:
(we will call this directory `apes`, but you can use a different name).

Reply:
thanks!